test/napi/napi_tsfn.test.js

```diff
@@ -0,0 +1,208 @@
+'use strict';
```

Comment:
Are this file a copy from https://github.com/Rokid/ShadowNode/blob/master/test/addons-napi/test_threadsafe_function/test.js? If yes, suggest that open an issue on [nodejs/napi-test-suite](https://github.com/nodejs/napi-test-suite) to track how should we address this with N-API team.